bridges/FeedReducerBridge.php

```diff
@@ -0,0 +1,62 @@
+<?php
+class FeedReducerBridge extends FeedExpander {
+
+	const MAINTAINER = 'mdemoss';
+	const NAME = 'Feed Reducer';
+	const URI = '';
+	const DESCRIPTION = 'Choose a percentage of a feed you want to see.';
+	const PARAMETERS = array( array(
+		'url' => array(
+			'name' => 'Feed URI',
+			'required' => true
+		),
+		'percentage' => array(
+			'name' => 'percentage',
+			'type' => 'number',
+			'exampleValue' => 50,
+			'required' => true
+		)
+	));
+	const CACHE_TIMEOUT = 3600;
+
+	public function collectData(){
+		if((preg_match('#^http(s?)://#i', $this->getInput('url')) === 1)) {
```

Comment:
There is no need to check the return value of `preg_match` explicitly. If there is no match it will either be `false` or `0`, both of which fail the check.

```suggestion
		if(preg_match('#^http(s?)://#i', $this->getInput('url'))) {
```